project/api/models.py

```diff
@@ -112,6 +127,13 @@ class AtomicPolygon(models.Model):
     children = models.ManyToManyField(
         "self", blank=True, symmetrical=False, related_name="parents"
     )
+    st = models.ForeignKey(
```

Comment:
This means that each AP belongs to only one STV, right? In this case it's incorrect. On AP can belong to multiple STV whose time intervals don't collide.

Reply:
Oh, right. Wouldn't a m2m field be correct in that case? With an added constraint to check for overlapping time intervals ofc

Reply:
Ah, you're totally right. Your previous implementation was correct, sorry about that.

Reply:
You're fine, the constraint was a nice catch :sweat_smile: